tests/legacy-cli/e2e/tests/experimental/ivy-lazy-load.ts

```diff
@@ -5,23 +5,88 @@
  * Use of this source code is governed by an MIT-style license that can be
  * found in the LICENSE file at https://angular.io/license
  */
-import { readdirSync } from 'fs';
-import { prependToFile, replaceInFile } from '../../utils/fs';
+import { prependToFile, replaceInFile, writeFile } from '../../utils/fs';
 import { ng } from '../../utils/process';
-import { createProject } from '../../utils/project';
+import { createProject, updateJsonFile } from '../../utils/project';
+import { expectToFail } from '../../utils/utils';
 
-export default async function() {
-  await createProject('ivy-project', '--enable-ivy');
+export async function ivyLazyLoadSetup(projectName: string) {
+  // Make Ivy project.
+  await createProject(projectName, '--enable-ivy');
 
+  // Add lazy route.
   await ng('generate', 'module', 'lazy', '--routing');
   await prependToFile('src/app/app.module.ts', `import { RouterModule } from '@angular/router';`);
```

Comment:
this can be removed if the above is done.

Reply:
Done